### ISSUE TITLE
Link Cable Priority Not Working With Storage Draws

### ISSUE THREAD

**Issue by ratkinsonuk** [PrinceOfAmber/Storage-Network#483]
Minecraft Version: 1.19

Forge Version: Unknown

Mod Version: 1.6.5

Single Player or Server: Single

Describe problem (what you were doing; what happened; what should have happened):
I'm playing the Seaopolis 2 pack, but having trouble setting the priority on a filtered link cable. I want to force everything into the Storage Draw network via a Storage Controller, so I've put the priority down to -30, but items are still being put into chests with have a link cable priority of zero.

Log file link: N/A

Video/images/gifs (direct upload or link): Available on Request

**Comment by SiriosDev** [comment]
yep, same here, it seems that the priority is not calculated correctly, cause by interacting with the priority selector it comes back to work

**Comment by Malkierian** [comment]
I think I just encountered this too. I was hooking a filtered link cable up to a Simple Storage Network controller, and I was setting it to -5 priority and Allow mode, then putting in the items I wanted, and they weren't going in. However, changing the priority of the link cable after having all the filters set updated the filter and everything started inserting (thanks for that Sirios). So it looks like the filter just isn't auto updating just from adding something to the filter. Was on 1.19.2 version 1.7.0.
